code/noRAM/app/src/main/java/com/example/noram/AttendeeAnnouncementsActivity.java

```diff
@@ -32,6 +42,31 @@ protected void onCreate(Bundle savedInstanceState){
 
         // retrieve corresponding event in database
         int eventID = getIntent().getIntExtra(eventIDLabel,0);
-        // TODO: Actually query database to get corresponding event
+
+        MainActivity.db.getEventsRef().document(String.valueOf(eventID)).get().addOnSuccessListener(documentSnapshot -> {
+            if (documentSnapshot.exists()) {
+                Event event = documentSnapshot.toObject(Event.class);
+                // displayAnnouncements(event);

```

Comment:
is this meant to be commented out

Reply:
yes it is but this stuff is slightly behind my branch so ignore it for now